src/frontend/js/pages/DashboardCreditCardsManagement/DashboardEditCreditCard.spec.tsx

```diff
@@ -97,6 +97,7 @@ describe('<DahsboardEditCreditCard/>', () => {
   });
 
   it('updates a credit card name', async () => {
+    fetchMock.get('https://endpoint.test/api/v1.0/user/me', richieUser);
```

Comment:
You could add it in the `beforeEach` nope ?

Reply:
it's not needed everywhere (not on the last test).